
function failureCall(error) {
    logDebug(failureCall.name,error);
    
    }
    